Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/TraceLocationCheckIn.kt

```diff
@@ -0,0 +1,21 @@
+package de.rki.coronawarnapp.eventregistration.checkins
+
+import org.joda.time.Instant
+
+interface TraceLocationCheckIn {
```

Comment:
I think we can simply call it `CheckIn`? TechSpec is also only calling it `CheckIn`

Reply:
Or let's call it chickens 🐓

Reply:
renamed it